sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -1,19 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
 
 ### Breaking Changes
 
-* Move header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`
+* Moved header `azure/storage/files/datalake/datalake.hpp` to `azure/storage/files/datalake.hpp`.
+* Moved returning model types and related functions in `Azure::Storage::Files::DataLake` to `Azure::Storage::Files::DataLake::Models`, and made other code private by moving it into `Azure::Storage::Files::DataLake::Details`.
+* Renamed `Azure::Storage::Files::DataLake::ServiceClient` to `Azure::Storage::Files::DataLake::DataLakeServiceClient`.
 
 ## 1.0.0-beta.4 (2020-10-16)
 
 ### Bug Fixes
```

Comment:
move bug fix below the features/breaking change section here too.

Reply:
How did I miss that. Thanks for the catch!